airbyte-integrations/connectors/source-twilio/source_twilio/manifest.yaml

```diff
@@ -0,0 +1,107 @@
+version: "0.29.0"
+
+definitions:
+  selector:
+    extractor:
+      field_path: []
+  requester:
+    url_base: "https://api.twilio.com"
+    http_method: "GET"
+    authenticator:
+      type: BasicHttpAuthenticator
+      username: "{{ config['account_sid'] }}"
+      password: "{{ config['auth_token'] }}"
+  datetime_cursor:
+    type: "DatetimeBasedCursor"
+    start_datetime:
+      datetime: "{{ config['start_date'] }}"
+      datetime_format: "%Y-%m-%d"
+    end_datetime:
+      datetime: "{{ now_utc() }}"
+      datetime_format: "%Y-%m-%d %H:%M:%S.%f+00:00"
+    step: "P1D"
+    datetime_format: "%Y-%m-%d"
+    cursor_granularity: "P1D"
+    cursor_field: "date"
+  retriever:
+    record_selector:
+      $ref: "#/definitions/selector"
+    paginator:
+      type: NoPagination
```

Comment:
@Alcadeus0 are you planning to add the paginator?

Reply:
@marcosmarxm Yep I have added it